clients/githubrepo/client.go

```diff
@@ -204,6 +204,17 @@ func (client *Client) GetCreatedAt() (time.Time, error) {
 	return client.repo.CreatedAt.Time, nil
 }
 
+func (client *Client) GetMFARequired() (required bool, err error) {
+	org, _, err := client.repoClient.Organizations.Get(context.Background(), client.repourl.owner)
```

Comment:
Could we add e2e tests of this on:
1. repo owned by a user
2. repo owned by an organization